libbeat/docs/security/user-access.asciidoc

```diff
@@ -2,21 +2,30 @@
 [[beats-user-access]]
 === Grant users access to {beatname_uc} indices
 
+:role_name: reader
+
 To enable users to access the indices {beatname_uc} creates, grant them `read`
 and `view_index_metadata` privileges on the {beatname_uc} indices. If they're
 using {kib}, they also need the `kibana_user` role.
 
-. Create a reader role that has the `read` and `view_index_metadata` privileges
+ifdef::apm-server[]
```

Comment:
Nit: I am wondering if it would be cleaner to define an `access_role` similar to how we define other vars (in the [index.asciidoc](https://raw.githubusercontent.com/elastic/apm-server/master/docs/index.asciidoc).
It would be set to `apm_user` for apm and to `{beat_default_index_prefix}_reader` for other beats. 

In this case you could check whether `access_role==apm_user`, for this distinction. 
It would also allow to replace  `{beat_default_index_prefix}_{role_name}` with `{access_role}`.

Reply:
YES, THIS IS BRILLIANT, THANK YOU!

Reply:
Gonna try to implement this now. Sorry for the all caps, I'm just really stoked on this idea. I couldn't for the life of me figure out how to clean this up.